Makefile

```diff
@@ -600,6 +619,9 @@ awx-kube-build: Dockerfile
 		--build-arg HEADLESS=$(HEADLESS) \
 		-t $(DEV_DOCKER_TAG_BASE)/awx:$(COMPOSE_TAG) .
 
+kind-dev-load: awx-kube-dev-build
```

Comment:
this change does not belong in this PR @djyasin can u revert this change

Reply:
Resolved with PR https://github.com/ansible/awx/pull/13770